### ISSUE TITLE
[CoE Starter Kit - BUG] Command Center Flow Status not Updating

### ISSUE THREAD

**Issue by Sinder88** [microsoft/coe-starter-kit#8037]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

When reviewing the COE Admin Command Center, under the CoE Flows tab, i notice a number of flows are showing as off and no last run status. When i go into the flow details, the flows are in fact on and have recent run history. 
<img width="958" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/84cbedfb-d7be-4215-ab5e-3c2c8c22640d">

<img width="671" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/dd396cf0-ac6c-4911-8a3a-37e332e08a2a">

<img width="940" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/797f70c2-d7b6-46a7-b467-134eb44da3d7">

<img width="958" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/41496bfc-c9be-4a97-a2ed-643160784327">

### Expected Behavior

Expect to see accurate data as to whether flow is on/off and when it last ran and status of run. 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

2.49

### What app or flow are you having the issue with?

COE Admin Command Center

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
I don't repro this. Can you please make sure you dont have unmanaged layers on the custom page? 
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/721f90eb-fe13-45ae-9977-fdb16a9e145f)


And then try to load again as well. 
There were some recent changes to one of the calls we make here and one of them got backed out. You may have experienced just an intermittent issue as a result.

**Comment by Sinder88** [comment]
Hi. Confirmed no unmanaged layer:
<img width="893" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/6445f60d-55c9-4a66-945b-da88a0cf7a67">

Seems it's showing last run but state is still = "Off". Wondering if State is actually referring to the flow being on or off or if i misunderstand. 

<img width="979" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/d28897b3-3f59-4573-8e29-55d1ad68a2c9">

<img width="1004" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/0abd2b98-0099-454d-8015-e2c0fb851786">

**Comment by Jenefer-Monroe** [comment]
Yes that is what it refers to, the state of the flow. I'll have to take a look and see if something changed. 
Can you please let me know what region your CoE envt is in as well as its type? This can be found in the product admin pages: https://admin.powerplatform.microsoft.com/
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/4c23e8c8-2f8b-4d8b-af43-79a9f289be69)

**Comment by Sinder88** [comment]
In fact, seems whenever i try turn a flow on/off from inside the COE toolkit (Command Center & Audit Start up guide for example), it doesn't actually turn the flow on/off. Wonder if the issue is related.

**Comment by Sinder88** [comment]
Region: United States
Type: Production

**Comment by Jenefer-Monroe** [comment]
Thank you yes those are likely related. And thats my region and type too so curious.
Do you notice any pattern on which ones are correct vs incorrect? This might help me repro.

**Comment by Sinder88** [comment]
I can't really pinpoint an exact pattern but i can see its seems to mainly be happening to inventory and environment request flows. 

<img width="962" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/1bf416d2-db18-4307-8b78-ef454442cd36">

<img width="838" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/109826106/10a037d0-dc1c-4cb6-bdbb-ab876a540a38">

**Comment by Jenefer-Monroe** [comment]
Will put on backlog to investigate

**Comment by Jenefer-Monroe** [comment]
to go with other changes form unpublished flows fix

**Comment by Jenefer-Monroe** [comment]
Moving all V2 deliverables to Dec release

**Comment by AmarSaiRam** [comment]
Dear User, 

We noticed that this issue has been pending for quite some time, and we have not received any recent updates from your end. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. 
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue. 
Thank you for your understanding and cooperation. 

Best Regards, 
CoE Starter Kit Team
